### PR TITLE
feat(storage): adjust filter for multi-currency

### PR DESCRIPTION
**Pull request by nduchak**
adjust offer filters (change title, change mapfrom/map-to transport)

### CODE REVIEW

File: src/components/pages/storage/buy/StorageOffersPage.tsx [hunk 1]
```diff
@@ -58,13 +58,14 @@ const StorageOffersPage: FC = () => {
...
-          divider=" "
-        />,
+        averagePrice: <ItemWUnit type="mediumPrimary" value={averagePrice.toString()} unit="USD" />,
+        // averagePrice: <CombinedPriceCell
+        //   price={averagePrice.toString()}
+        //   priceFiat={(averagePrice * rate).toString()}
+        //   currency="USD"
+        //   currencyFiat={currentFiat.displayName}
+        //   divider=" "
+        // />,
```
Comment (farinalvaro) [inline src/components/pages/storage/buy/StorageOffersPage.tsx:68]: Remove commented code

File: src/components/pages/storage/buy/StorageOffersPage.tsx [hunk 2]
```diff
@@ -47,7 +47,7 @@ const StorageOffersPage: FC = () => {
         id, system, availableSizeGB, averagePrice, subscriptionOptions,
       } = item
 
-      const { rate, displayName } = crypto.rbtc // FIXME: remove hard-coded currency
+      // const { rate, displayName } = crypto.rbtc // FIXME: remove hard-coded currency
```
Comment (farinalvaro) [inline src/components/pages/storage/buy/StorageOffersPage.tsx:50]: Remove comment

File: src/api/rif-marketplace-cache/storage/offers.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line max-classes-per-file
```
Comment (farinalvaro) [inline src/api/rif-marketplace-cache/storage/offers.ts:1]: Could it be better to extract `AvgBillingPriceService` class to a new file?

Comment (nduchak) [reply src/api/rif-marketplace-cache/storage/offers.ts:1]: Probably yes 👍

File: src/components/pages/storage/buy/StorageOffersPage.tsx (hunk 1 above)
Comment (nduchak) [reply src/components/pages/storage/buy/StorageOffersPage.tsx]: done

File: src/api/rif-marketplace-cache/storage/offers.ts (hunk 3 above)
Comment (nduchak) [reply src/api/rif-marketplace-cache/storage/offers.ts]: done

File: src/components/pages/storage/buy/StorageOffersPage.tsx (hunk 2 above)
Comment (nduchak) [reply src/components/pages/storage/buy/StorageOffersPage.tsx]: done

File: src/context/Services/storage/OffersContext.tsx [hunk 4]
```diff
@@ -142,7 +148,7 @@ export const StorageOffersContextProvider = ({ children }) => {
                 text: 'Error while fetching filters. ',
               })
             }),
-          api.fetchPriceLimits()
+          apiAvgBillingPrice.fetchPriceLimits()
```
Comment (farinalvaro) [inline src/context/Services/storage/OffersContext.tsx:151]: We need to add `apiAvgBillingPrice` to the array dependencies of this `useEffect`

Comment (nduchak) [reply src/context/Services/storage/OffersContext.tsx:151]: Done

**Review by farinalvaro — APPROVED** [review]
LGTM

**Pull request merged**